corehq/apps/hqmedia/models.py

```diff
@@ -713,24 +713,27 @@ def all_media(self, lang=None):
         media = copy(self.menu_media(self, lang=lang))
 
         # Form questions
-        parsed = self.wrapped_xform()
-        if parsed.exists():
-            try:
+        try:
+            parsed = self.wrapped_xform()
+            if parsed.exists():
                 self.validate_form()
-            except (XFormValidationError, XFormException):
+            else:
                 return media
```

Comment:
Does adding this `else: return media` here change behavior beyond error handling? I'm looking at the "ignore whitespace" diff

Reply:
It does not: the rest of the function was inside an `if parsed.exists():` block, so now I'm just returning early and moving the later code out of that if block.